CHANGELOG.md

```diff
@@ -1,7 +1,21 @@
 # Changelog
 
 ## [Unreleased]
+
+## [0.18.0] - 2024-08-05
+
+Usability
 - Add attributes `seek_from_start`, `seek_from_current`, `seek_from_end`, and `seek_rewind` to control the position of the reader before reading a field ([#360](https://github.com/sharksforarms/deku/pull/360))
```

Comment:
This wasn't you, this was me. However, we should mention that `Seek` is now required.